.travis.yml

```diff
@@ -11,6 +11,7 @@ before_script:
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
 script:
+- make test
```

Comment:
Could do just a `make lint` here, as `nosetests` below will also run tests + generate a report

Reply:
Done! Thanks for the suggestion!